### DISCUSSION TITLE
go-bindata command: command not found err during GoLang development

### DISCUSSION THREAD

**Discussion by toshikijp** [andatoshiki/toshiki-discuss#20] [Q&A] [answered]
As the Question describes

**Comment by andatoshiki** [comment, marked as answer]
To perform the installation with modern versions of Go, you should run:

go install github.com/go-bindata/go-bindata/go-bindata@latest
This command will compile and put the binary into the GOPATH\bin folder on your machine, so if you want to be able to run it from anywhere, you should add GOPATH\bin to your PATH.

And then try and run it:

go-bindata
It's a bit awkward repetition, because of the project directory structure. The first go-bindata is github username, the second is the project name and the third is where the main package of the program is to be found.
